code/train_LSTM.py

```diff
@@ -273,8 +265,9 @@ def LSTM_Model(X):
   print("Splitting dataset into training test...")
   X_train_entire, X_validation_entire, y_train_entire, y_validation_entire = train_test_split(all_examples, all_labels, test_size=FLAGS.test_size)
   print("Creating Recurrent LSTM model...")
-  lstm_model = LSTM_Model(X_train_entire)
+  lstm_model = LSTM_Model()
```

Comment:
Why is this changing?

Reply:
No need to pass as a param since we can necessary values from vggish_params.py